pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -128,98 +101,82 @@ social:
             %button.ctabutton-white
               = hoc_s(:hoc_live_learn_more)
 
-    %br/
-    %br/
-
   -# Description
-
-  #description.row
-    .col-xs-1
-      &nbsp;
-    .col-xs-10.hoc-description
-      .number-served= hoc_s(:front_join_us_n_students_served).gsub("#", format_integer_with_commas(fetch_hoc_metrics['started']).to_s)
-      = hoc_s(:front_intro_default_2)
-      %br
-      = view :hoc_events_counter
-
-  %br/
-  %br/
+  %section.hoc-description
+    .wrapper
+      %h2.number-served=hoc_s(:front_join_us_n_students_served).gsub("#", format_integer_with_commas(fetch_hoc_metrics['started']).to_s)
+      %p=hoc_s(:front_intro_default_2)
+      %p=view :hoc_events_counter
 
   -# Map
-  - unless @country == 'ma' # Hide map for Morocco because of sensitive border disputes misrepresented in map
-    #maparea.full-width
-      -# Show a picture of the map unless it is currently the week of HOC.
-      -if hoc_mode != "actual-hoc"
-        = view :hoc_events_map
-        .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
-      -else
-        %a{href: resolve_url('/map')}
-          %img{src: '/images/fit-2276/map.jpg', style: 'width: 100%;'}
-        .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
-
-  -# Signup and Highlights
-
-  -# Show sign up form above highlights when we're trying to increase sign ups. Otherwise, feature highlights first which is more engaging and evergreen.
-  -if ["pre-hoc", "soon-hoc"].include?(hoc_mode)
-    = view :home_signup, hoc_mode: hoc_mode
-    = view :home_highlights
-  -else
-    = view :home_highlights
-    = view :home_signup, hoc_mode: hoc_mode
+  %section.map
+    .wrapper
+      - unless @country == 'ma' # Hide map for Morocco because of sensitive border disputes misrepresented in map
+        #maparea.full-width
+          -# Show a picture of the map unless it is currently the week of HOC.
+          -if hoc_mode != "actual-hoc"
+            = view :hoc_events_map
+            .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
+          -else
+            %a{href: resolve_url('/map')}
+              %img{src: '/images/fit-2276/map.jpg', style: 'width: 100%;'}
+            .footnote!=hoc_s(:map_warning_markdown, markdown: :inline, locals: {events_url: resolve_url('/events')})
+
+  -# Join us
+  %section#join.join-us
+    .wrapper
+      %h2=hoc_s(:hoc_live_join_us)
+      %h3=hoc_s(:hoc_homepage_join_the_movement_description)
+      -if ["pre-hoc", "soon-hoc"].include?(hoc_mode)
+        = view :home_signup, hoc_mode: hoc_mode
+
+  -# What is HoC?
+  -# This section remains evergreen, and the video no longer moves next to the signup form during different hoc_modes.
+  %section.what-is-hoc
+    .wrapper
+      %h2=hoc_s(:front_what_is_hoc)
+      %h3=hoc_s(:hoc_homepage_what_is_hoc_subhead)
+      =view :index_video
```

Comment:
This isn't showing up for me. Is there possibly something wrong with my environment?

<img width="961" alt="Screen Shot 2022-09-23 at 8 44 28 AM" src="https://user-images.githubusercontent.com/46464143/191962978-85dbf697-6b77-4ebb-a113-bd6b32489ffb.png">

Reply:
It's not showing up for me either on localhost, so I superimposed a screenshot of the map into the screenshot (it doesn't show on `staging` either). I assumed it's bc of the mapbox api not working in a dev environment? I also don't know much about setting up local environments though!

Reply:
I'm thinking of the video, not the map but the video doesn't show up for me on staging either so this seems fine

Reply:
Ack, got my sections confused! I'm also not seeing either the map or video on staging.

Reply:
On an older enlistment, I'm seeing an embedded youtube video play fine on my localhost, so I'd be slightly concerned if you're not seeing it.  I haven't heard of videos not working on staging either.. I think they should work on all environments.

For the map, if you open a developer console, you might see an error about needing an access token?  It does seem worth getting that working locally as well, to make sure that there aren't any surprises when this gets to production.

Reply:
I'm seeing the video on staging now — I wonder if it had something to do with the staging build failing yesterday? ([Slack convo link](https://codedotorg.slack.com/archives/C0T0PNTM3/p1663951495015769))